spec/javascript/components/supplemental_application/__snapshots__/SupplementalApplicationPage.test.js.snap

```diff
@@ -1434,7 +1434,7 @@ Array [
                         onBlur={[Function]}
                         onChange={[Function]}
                         onFocus={[Function]}
-                        value="Yes"
```

Comment:
Curious why this changed.

Reply:
The initial value used to be based on the monthly rent, so if you started with:
monthly_rent = $100
bmr_parking_space_assigned = "No" (or null)

The old version would initialize to "Yes" and the new version inits to "No"